modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -286,6 +284,14 @@ def claims_load_testing
         def mocking
           Settings.claims_api.benefits_documents.use_mocks
         end
+
+        def find_claim(claim)
+          # Fetch the claim by md5 if it doesn't have an ID (given duplicate md5)
+          if claim.id.nil? && claim.errors.find { |e| e.attribute == :md5 }&.type == :taken
+            claim = ClaimsApi::AutoEstablishedClaim.find_by(md5: claim.md5) || claim
+          end
+          ClaimsApi::AutoEstablishedClaim.find_by(header_hash: claim.header_hash) || claim if claim.nil?
```

Comment:
I don't think works as intended, but the specs don't actually cover this line so it's impossible to tell. Will investigate.